Added Additional Information About Wish Lists Managing (#1378)
magento/merchdocs: src/customers/account-dashboard-my-wish-list.md

    ![]({% link images/images/account-dashboard-wishlist-sharing-email.png %}){: .zoom}
    _Shared Wish List Email_

{:.ee-only}
## Move items to a different list

> 1. Select the checkbox of each item to be moved.

Reviewer: ```suggestion
1. Customer selects the checkbox of each item to be moved.
```

> 1. Click **Move Selected to Wish List** and do one of the following:

Reviewer: ```suggestion
1. Clicks **Move Selected to Wish List** and does one of the following:
```

>     - Choose an existing wish list.
>     - Click **Create New Wish List**.

Reviewer: ```suggestion
    - Chooses an existing wish list.
    - Clicks **Create New Wish List**.
```

{:.ee-only}
## Copy items to a different list

> 1. Select the checkbox of each item to be moved.

Reviewer: ```suggestion
1. Selects the checkbox of each item to be moved.
```

> 1. Click **Copy Selected to Wish List** and do one of the following:

Reviewer: ```suggestion
1. Clicks **Copy Selected to Wish List** and does one of the following:
```

>     - Choose an existing wish list.
>     - Click **Create New Wish List**.

Reviewer: ```suggestion
    - Chooses an existing wish list.
    - Clicks **Create New Wish List**.
```

{:.ee-only}

> ## Delete Created Wish Lists

Reviewer: ```suggestion
## Delete an existing wish list
```

> ## Delete Created Wish Lists

Reviewer: ```suggestion
## Delete created wish lists
```

> 1. Open the wish list to be deleted.

Reviewer: ```suggestion
1. Customer opens the wish list to be deleted.
```

> 1. Click the **Delete Wish List** button.

Reviewer: ```suggestion
1. Clicks <span class="btn">Delete Wish List</span>.
```

> 1. When prompted to confirm, click <span class="btn">OK</span> button.

Reviewer: ```suggestion
1. When prompted to confirm, clicks <span class="btn">OK</span>.
```
